main.js

```diff
@@ -11,10 +12,29 @@ if (PapanUtils.isElectron()) {
 const mainNode = require('./src/server/main-node.js')
 mainNode.main()
 
+function readJSON (filename) {
+  return new Promise((resolve, reject) => {
+    fs.readFile(filename, (err, data) => {
+      if (err) resolve({})
+      resolve(JSON.parse(data))
+    })
+  })
+}
+
 if (argv.auth_server) {
-  let config = {}
-  const express = require('express')
-  const papanAuth = require('./src/server/auth/server.js')
-  let app = express()
-  papanAuth.registerServer(app, config).then(app.listen(8081))
+  let googleAuthConfig = {}
+  readJSON('config/google-auth-config.json').then((googleAuthConfigRead) => {
+    googleAuthConfig = googleAuthConfigRead
+    return readJSON('config/pg-config.json')
+  }).then((pgConfig) => {
```

Comment:
Ah, I was wondering about this. That's good to know. This codepath is mainly for debugging the auth server in a desktop environment - the real Auth server that's currently running at https://auth.papan.online needed some specific additional candy wrapper to make sure the openshift container is happy and can stop, start, scale, and do health checking. I'll eventually publish that code, but right now it's in the private openshift repository since it contains credentials.